pkg/pager/err.go

```
@@ -0,0 +1,5 @@
+package pager
+
+type errMsg struct{ err error }
```

golangci-lint (1.64.8)

[error] type `errMsg` is unused

(unused)